src/main/java/at/hannibal2/skyhanni/config/features/About.java

```diff
@@ -26,6 +26,7 @@ public class About {
 
     @ConfigOption(name = "Used Software", desc = "Information about used software and licenses")
     @Accordion
+    @Expose
```

Comment:
Do we really need to save this into the config? we could refetch it from the jar on every startup anyway

Reply:
its just gonna save an empty object, and this is there to stop the warning as well